Fix broken TensorRT-LLM link to deepseekv3

fernandaspets opened the pull request:
Fix broken TensorRT-LLM link to deepseekv3 examples

fernandaspets committed:
Fix broken TensorRT-LLM link to deepseekv3

Fix broken TensorRT-LLM link to deepseekv3

yangaxnkohla commented:
Hi fernandaspets, the link https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/deepseek_v3 is still broken, could you please fix that too

fernandaspets committed:
Fix link to DeepSeek-V3 model in README

Updated the text inside the square brackets [ ] to accurately describe the destination

Ensured the URL inside the parentheses ( ) is the correct one

fernandaspets commented:
> Hi fernandaspets, the link https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/deepseek_v3 is still broken, could you please fix that too

yes now I have updated the text inside the square brackets [ ] to accurately describe the destination
